tools/labs/templates/networking/5-udp-sock/udp_sock.c

```diff
@@ -114,15 +114,13 @@ int __init my_udp_sock_init(void)
 	return 0;
 
 out_release:
-	/* TODO 1/0: cleanup socket */
 	sock_release(sock);
 out:
 	return err;
 }
 
 void __exit my_udp_sock_exit(void)
 {
-	/* TODO 1/1: cleanup socket */
```

Comment:
This should not be deleted. See the above comment for context.

Reply:
I've solved this in the newest commit.